pyproject.toml

```diff
@@ -101,3 +101,13 @@ coverage = ["test --cov=acoular"]
 platform.linux.pre-install-commands = ['cat /proc/cpuinfo']
 platform.macos.pre-install-commands = ['sysctl -a machdep.cpu']
 platform.windows.pre-install-commands = ['systeminfo']
+
+
+[tool.hatch.build.targets.sdist]
+exclude = [
```

Comment:
Only what is needed to build the package should be included in the sdist. I think that is just ./acoular, pyproject.toml and some other files in the project root (approx 150 kB). The last sdist on pypi includes everything including test data and is stunning 7 MB large.

Reply:
We handled it differently across various releases (24.03, 23.11, 23.06 -> approx. 7 MB, 22.03, 21.05, ... < 1 MB).

Reply:
now includes only the source directory